README.md

```diff
@@ -88,7 +150,9 @@ Algorithms not exposed (and reason why)
 * ECDH-ES+A128KW (Chrome and IE currently don't support it)
 * ECDH-ES+A192KW (Chrome and IE currently don't support it)
 * ECDH-ES+A256KW (Chrome and IE currently don't support it)
-* RSA-PSS, ECDSA (could be added once Chrome supports these)
+* ECDSA (could be added once Chrome supports these)
+* RSA-PSS on Linux, OSX and iOS (Chrome supports it only on Windows)
```

Comment:
I didn't found any info about that, but that's what chromium devs declare on https://www.chromium.org/blink/webcrypto and what my tests confirm